prowler/lib/scan/scan.py

```diff
@@ -239,6 +239,20 @@ def scan(
             Exception: If any other error occurs during the execution of a check.
         """
         try:
+            # Load bulk compliance frameworks
```

Comment:
Instead of doing this two times: one here and another one during the `__init__` maybe you can store it in the class. What do you think?